src/sentry/search/snuba/executors.py

```diff
@@ -1215,19 +1213,39 @@ def get_basic_event_snuba_condition(
         if not isinstance(item, list):
             raw_conditions = [raw_conditions]
 
+        query_builder = self.def_get_query_builder(joined_entity)
+        query_builder.default_filter_converter(search_filter)
+
+        conver_value_to_date = False
         output_conditions = []
         for item in raw_conditions:
-            column_name = item[0]
-            # do some name mapping for snuba
-            column_name = column_name.replace("stack.", "stacktrace.")
-            if ATTR_CHOICES.get(column_name) is not None:
-                raw_column = ATTR_CHOICES.get(column_name)
-                column_name = raw_column.value.event_name
-
-            column = Column(column_name, joined_entity)
+            lhs = item[0]
+            if isinstance(lhs, str):
+                raw_column = map_field_name_from_format_search_filter(lhs)
+                lhs = query_builder.resolve_column(raw_column)
+            else:
+                # need to convert dates to timestamps
```

Comment:
why do we need to do this timestamp business?

Reply:
@JoshFerge because we need to convert from a UNIX timestamp to datetime

Reply:
why? do we do this in our other query executors?

Reply:
@JoshFerge I realize now that we already handle the date filter with `calculate_start_end` so I can remove that special logic, we just skip that filter